lib/jsonapi/serializer.ex

```diff
@@ -144,7 +144,7 @@ defmodule JSONAPI.Serializer do
   end
 
   defp merge_links(doc, data, view, conn, false, true) do
-    pagination_links = view.links(data, conn)
+    pagination_links = view.links(data, conn, doc[:meta])
```

Comment:
Can we do `Map.get(doc, :meta, %{})` so we have a default here in case meta is nil.